### PR TITLE
Fix millis issues.

### PR DESCRIPTION
**Pull request by AureliaDolo**

### CODE REVIEW

**Review by AurelienFT — APPROVED** [review]

**Comment by AureliaDolo** [comment]
We should really bring back the test run node test

**Review by adrien-zinger — DISMISSED** [review]

**Comment by AureliaDolo** [comment]
bors merge

**Comment by adrien-zinger** [comment]
> We should really bring back the test run node test

from => from_millis!!! :slightly_smiling_face: naming naming naming

File: massa-time/src/lib.rs [hunk 1]
```diff
@@ -342,7 +342,7 @@ impl MassaTime {
     /// assert_eq!(massa_time.to_utc_string(), "2022-01-01T00:00:00Z")
```
Comment (adrien-zinger) [inline massa-time/src/lib.rs:342]: 1_640_995_200_000

**Review by AurelienFT — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**